src/main/java/ladder/domain/InputResult.java

```diff
@@ -0,0 +1,30 @@
+package ladder.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class InputResult {
+	private final List<String> inputResults;
+
+	public InputResult(String inputResults, int playerSize) {
+		validateInputResult(inputResults, playerSize);
+		this.inputResults = Arrays.stream(inputResults.split(","))
+			.map(inputResult -> inputResult.trim())
```

Comment:
method reference를 사용해보면 어떨까요? 😄 
```suggestion
			.map(String::trim)
```